apps/central-scan/backend/src/importer.ts

```diff
@@ -224,7 +225,9 @@ export class Importer {
           adjudicationReasons: store.getAdjudicationReasons(),
           markThresholds,
           allowOfficialBallotsInTestMode,
-          minimumDetectedScale,
+          minimumDetectedScale:
+            minimumDetectedBallotScaleOverride ??
+            DEFAULT_MINIMUM_DETECTED_BALLOT_SCALE,
```

Comment:
I considered pushing this logic even further down the stack into the Rust code, putting both the default and override handling there. But decided to keep this change minimal for lack of time.

Handling at this level works pretty well, too, since TS reminds you to convert from the `minimumDetectedBallotScaleOverride` system setting to the `minimumDetectedScale` that the interpreter's public interface expects.

Reply:
Seems fine for now!